hack/local-development/local-garden/certificates/certs/.gitignore

```diff
@@ -1 +0,0 @@
-gardenlet-*.crt
```

Comment:
Can you explain why this .gitignore is removed?

Reply:
See https://github.com/gardener/gardener/pull/5115#discussion_r771517964